.JamesLau-fe-project/src/Components/ArticleContainer.jsx

```diff
@@ -0,0 +1,26 @@
+import { useState, useEffect } from "react";
+import { getArticles } from "../api";
+import ArticleCard from "./ArticleCard";
+
+const ArticleContainer = () => {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    getArticles().then((data) => {
+      setArticles(data);
+    });
+  }, []);
+
+  return (
+    <>
```

Comment:
we don't need to use fragments if there is a parent tag, which the div is, it wraps everything else.